ironfish/src/consensus/consensus.ts

```diff
@@ -28,6 +28,11 @@ export type ConsensusParameters = {
    * Max block size
    */
   maxBlockSizeBytes: number
+
+  /**
+   * The minimum fee that a transaction must have to be accepted
+   */
+  minFee: number
```

Comment:
This is an amount in ore so it should not be a number

Reply:
leaving as is due to issues with yup supporting `BigInt`